### PR TITLE
Add spotsAvailable calculation for Events

### PR DESCRIPTION
**Pull request by connernilsen**
This is the work for [this ticket](https://trello.com/c/RvTyNTGG/131-llb-add-capacity-field-to-event-apis). 

[Api docs](https://github.com/Code-4-Community/dev-spikes/blob/master/api-spec/llb/api-spec.md) mention a lot of other endpoints that may be relevant, but they're not available in master yet so I skipped them for now. All that's available in the `EventsProcessor` now are the routes to create an event and get a single event (unless I'm completely missing something).

Also, this has a minor update to one of the comments in the Dockerfile. It should be noted that the `EXPOSE` Dockerfile keyword is basically only used as documentation for the person running the container that the given port should be port-forwarded by the user. See the [docker documentation](https://docs.docker.com/engine/reference/builder/#expose).
>The `EXPOSE` instruction does not actually publish the port. It functions as a type of documentation between the person who builds the image and the person who runs the container, about which ports are intended to be published. To actually publish the port when running the container, use the `-p` flag on docker run to publish and map one or more ports, or the `-P` flag to publish all exposed ports and map them to high-order ports.

### CODE REVIEW

File: service/src/main/java/com/codeforcommunity/processor/EventsProcessorImpl.java [hunk 1]
```diff
@@ -64,4 +66,16 @@ private EventsRecord eventRequestToRecord(CreateEventRequest request) {
...
   }
+
+  /**
+   * Returns the count of spaces available for the given event.
+   *
+   * @param event the Events object to find get the spaces available for.
+   * @return an int representing the number of available spaces.
+   */
+  private int getSpotsAvailable(Events event) {
+    Result<UserEventsRecord> userEventsRecord = db.selectFrom(USER_EVENTS)
```
Comment (liammoyn) [inline service/src/main/java/com/codeforcommunity/processor/EventsProcessorImpl.java:77]: Use a count sql statement. The database is optimized to do things quickly so the less we can do in Java code the better. @LukeBoyer Might have already made this query btw so be sure to message him

Comment (LukeBoyer) [reply service/src/main/java/com/codeforcommunity/processor/EventsProcessorImpl.java:77]: yeah, I have that exact same logic in mine

Comment (LukeBoyer) [reply service/src/main/java/com/codeforcommunity/processor/EventsProcessorImpl.java:77]: but with count

Comment (connernilsen) [reply service/src/main/java/com/codeforcommunity/processor/EventsProcessorImpl.java]: fixed

**Pull request merged**